src/System.Runtime.Intrinsics.X86/ref/System.Runtime.Intrinsics.X86.cs

```diff
@@ -780,24 +813,27 @@ public static class Sse
         public static Vector128<float> MoveHighToLow(Vector128<float> left,  Vector128<float> right) { throw null; }
         public static Vector128<float> MoveLowToHigh(Vector128<float> left,  Vector128<float> right) { throw null; }
         public static Vector128<float> MoveScalar(Vector128<float> upper, Vector128<float> value) { throw null; }
+        public static int MoveMask(Vector128<float> value) { throw null; }
         public static Vector128<float> Multiply(Vector128<float> left,  Vector128<float> right) { throw null; }
         public static Vector128<float> MultiplyScalar(Vector128<float> left, Vector128<float> right) { throw null; }
         public static Vector128<float> Or(Vector128<float> left,  Vector128<float> right) { throw null; }
         public static Vector128<float> Reciprocal(Vector128<float> value) { throw null; }
         public static Vector128<float> ReciprocalScalar(Vector128<float> value) { throw null; }
         public static Vector128<float> ReciprocalSqrt(Vector128<float> value) { throw null; }
         public static Vector128<float> ReciprocalSqrtScalar(Vector128<float> value) { throw null; }
-        public static Vector128<float> Set(float e3, float e2, float e1, float e0) { throw null; }
-        public static Vector128<float> Set1(float value) { throw null; }
+        public static Vector128<float> SetVector128(float e3, float e2, float e1, float e0) { throw null; }
+        public static Vector128<float> SetAllVector128(float value) { throw null; }
         public static Vector128<float> SetScalar(float value) { throw null; }
-        public static Vector128<float> SetZero() { throw null; }
+        public static Vector128<float> SetZeroVector128() { throw null; }
         public static Vector128<U> StaticCast<T, U>(Vector128<T> value) where T : struct where U : struct { throw null; }
         public static Vector128<float> Shuffle(Vector128<float> left, Vector128<float> right, byte control) { throw null; }
         public static Vector128<float> Sqrt(Vector128<float> value) { throw null; }
         public static Vector128<float> SqrtScalar(Vector128<float> value) { throw null; }
         public static unsafe void StoreAligned(float* address, Vector128<float> source) { throw null; }
         public static unsafe void StoreAlignedNonTemporal(float* address, Vector128<float> source) { throw null; }
         public static unsafe void Store(float* address, Vector128<float> source) { throw null; }
+        public static unsafe void StoreHigh(float* address, Vector128<float> source) { throw null; }
+        public static unsafe void StoreLow(float* address, Vector128<float> source) { throw null; }        
```

Comment:
StoreLow from Sse2 looks to be missing

Reply:
https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Intrinsics.X86/ref/System.Runtime.Intrinsics.X86.cs#L1065

That was not missing, so it is not shown in this PR.

Reply:
:+1: